tox.ini

```diff
@@ -24,3 +25,13 @@ skip_install = true
 commands = isort --recursive --check-only --diff
 deps = isort
 skip_install = true
+
+[testenv:mypy]
+commands =
+    mypy \
+        --python-version 3.6 \
+        --ignore-missing-imports \
+        --disallow-untyped-defs \
```

Comment:
Maybe run mypy in [`strict` mode](https://mypy.readthedocs.io/en/stable/command_line.html#miscellaneous-strictness-flags) rather than specifying error-checking flags?